packages/combobox/src/Combobox/Combobox.styles.ts

```diff
@@ -175,13 +173,31 @@ export const comboboxDisabledStyles: Record<Theme, string> = {
   `,
 };
 
-export const comboboxErrorStyles: Record<Theme, string> = {
-  [Theme.Light]: css`
-    border-color: ${palette.red.base};
-  `,
-  [Theme.Dark]: css`
-    border-color: ${palette.red.light1};
-  `,
+export const comboboxStateStyles: Record<State, Record<Theme, string>> = {
+  [State.error]: {
+    [Theme.Light]: css`
+      border-color: ${palette.red.base};
```

Comment:
we have border color tokens for this WHICH i am realizing are misspelled (there's a missing `e` in `lightMod`) - we should fix this and use tokens here

Reply:
typo is fixed here: https://github.com/mongodb/leafygreen-ui/pull/2318/commits/ae8defc476d8e65c7766c85a599f0ab6c35c0739

applied the appropriate color tokens where possible :)